Article/Article.js

```diff
@@ -1,20 +1,23 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// this.expandButton = document.querySelectorAll('.expandButton');
+// expandButton.forEach(expandButton => expandButton.textContent = "expand");
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
```

Comment:
I recommend maybe renaming this to something more around what the actual element you will be touching actually is rather then a generic name like domElemet. It helps to visualize what you're grabbing.